Tools/interface/Year.h

```diff
@@ -1,7 +1,7 @@
 #ifndef JMTucker_Tools_interface_Year_h
 #define JMTucker_Tools_interface_Year_h
 
-#define MFVNEUTRALINO_2017
+#define MFVNEUTRALINO_2018
```

Comment:
In the main branch, we should probably leave this as its old default of 2017 (it's easy to remember to change it from 2017 to 2018 w/ our existing instructions, but maybe hard to remember the other way!).